charts/k0s/values.yaml

```diff
@@ -223,4 +223,23 @@ securityContext:
 # to manage service endpoints
 openshift:
   enable: false
-  
\ No newline at end of file
+
+isolation:
+  enabled: false
+
+  resourceQuota:
```

Comment:
please add an option `isolation.resourceQuota.enabled: true` so that people could disable the resource quota if wanted